Fix: make sure `e` or `w` are at the start of the line

T-727 opened the pull request:
This prevents commands which do not start with `e` or `w` from triggering the source
Example cases: `:execute` and `:Telescope`

T-727 committed:
Change COMMAND_SHORTCUT

make sure the `e` or `w` are at the start of the line

tzachar commented:
I dont think this is the correct fix.
`\%^` just means `start of file`, which is equivalent to changing the regex here to `^\%(e\|w\)\s\+`.

Can you please change to the above?

T-727 commented:
It should be practically the same since `\%^` matches the start of the string when used with strings but sure I'll change :)

T-727 committed:
Update init.lua

change requested in https://github.com/tzachar/cmp-fuzzy-path/issues/6#issuecomment-1290066768

tzachar commented:
10x
